sdk/testcontainers-vault/tsconfig.build.json

```diff
@@ -1,7 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "dist"
+    "outDir": "dist",
+    "experimentalDecorators": false,
```

Comment:
Why mention explicitly? same question for emitDecoratorMetadata

Reply:
I notice that it affect the bundle size. Typescript adds extra helpers when these are on.
We globally want it to be turned on since IDEs used the root tsconfig but when we build we turn it off for modules that don't use it.